setup-dev-env.sh

```diff
@@ -17,7 +17,8 @@ if ! grep -qF "DATABASE_USERNAME" .env; then
 fi
 
 if ! grep -qF "DATABASE_PASSWORD" .env; then
-	read -p 'Enter DATABASE_PASSWORD (leave blank to not set a password): ' psswd
+	read -s -p 'Enter DATABASE_PASSWORD (leave blank to not set a password): ' psswd
+	echo ""
```

Comment:
Just curious. I assume the echo "" is for readability when running the script. Is it to provide visual separation to the DATABASE environment variables from JWT, API_KEY_SECRET, FQDN/Hostname/IP Address, and SSL Self-Signed Cert section?

Reply:
Talked it out over call, but basic answer is yes since some of the 'read' flags make it so that newlines aren't applied properly.